index.js

```diff
@@ -15,6 +15,12 @@ module.exports = opts => {
 	opts = opts || {};
 
 	return psList().then(list => {
+		if (opts.killBrowser) {
```

Comment:
Open an issue before adding new options. I think you should use [`fkill`](https://github.com/sindresorhus/fkill) directly if you want to do this.

Reply:
Agreed. That is outside the scope of this tool.